Makefile

```diff
@@ -149,6 +149,14 @@ tc-image: container
 	@cp receptor packaging/tc-image/
 	@$(CONTAINERCMD) build packaging/tc-image -t receptor-tc
 
+receptorctl-test-venv: receptor
```

Comment:
The make target here should be a file rather than a directory (I would suggest `receptorctl-test-venv/bin/pytest`) so the Makefile can avoid rebuilding when not necessary.